### DISCUSSION TITLE
如何让v语言快速的抢占rust/c++/golang的市场

### DISCUSSION THREAD

**Discussion by putao0525** [vlang/v#17697] [Installation Issues]
1. 语法要稳定
2. 第三方的包要丰富
3. 要有有钱的公司推广
4. 有几个无法撼动的业务方向

**Comment by MatejMagat305** [comment]
you are right, but I think, the rich third part library come slow with trying many people 
by the way, Vlang is only language which knows build apk on termux, then send code to computer and compile again, it would be nice to find other speciel features whict Vlang can do until Vlang is young

**Comment by Wajinn** [comment]
Perhaps it would have been nice to also use Google or DeepL translation, as likely many can't read Chinese.

> 如何让v语言快速的抢占rust/c++/golang的市场 #17697
> 
> 语法要稳定
> 第三方的包要丰富
> 要有有钱的公司推广
> 有几个无法撼动的业务方向

How to make v-language grab the rust/c++/golang market fast #17697

The syntax should be stable
The third-party package should be rich
There should be rich companies to promote
There are several untouchable business directions

**Comment by getfile** [comment]
现在有大量的bug需要修复, 要达到稳定还需要很长时间.
感觉bug修复速度有点慢

**Reply by putao0525** [reply]
不太好招开发人员
